dcos_installer/upgrade.py

```diff
@@ -71,6 +71,25 @@
     exit 1
 fi
 
+# Check if the node has node/cluster checks and run them
+if 3dt check; then
```

Comment:
Nothing else calls `3dt check` except for this script. If that command exits 1, the only symptom would be this script silently skipping checks before running an upgrade, potentially breaking a cluster.